src/java/org/apache/cassandra/tools/nodetool/clustermetadata/CMSCmdGroupHelp.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.tools.nodetool.clustermetadata;
+
+import io.airlift.airline.Help;
+import org.apache.cassandra.tools.INodeProbeFactory;
+import org.apache.cassandra.tools.NodeTool;
+import org.apache.cassandra.tools.Output;
+
+/**
+ * Help class for Cluster Metadata Service nodetool command group.
+ */
+public class CMSCmdGroupHelp extends Help implements NodeTool.NodeToolCmdRunnable
```

Comment:
I like this ... this will print help when `nodetool cms` is executed as it just expects some other subcommand to be specified. This is not the same as for the case of repair_admin, when `nodetool repair_admin` is executed, it will default to `ListCmd`. We might theoretically default to "cms describe" if only `nodetool cms` is run.